packages/ag-charts-website/src/content/docs/high-performance-charts/_examples/line-series/data.ts

```diff
@@ -0,0 +1,30 @@
+const startPrice = 100;
+const maxDailyPriceChange = 5;
+const days = 1000000;
+
+// https://github.com/mikolalysenko/hash-int
+const A = new Uint32Array(1);
+function hashInt(x: number) {
+    A[0] = x | 0;
+    A[0] -= A[0] << 6;
+    A[0] ^= A[0] >>> 17;
+    A[0] -= A[0] << 9;
+    A[0] ^= A[0] << 4;
+    A[0] -= A[0] << 3;
+    A[0] ^= A[0] << 10;
+    A[0] ^= A[0] >>> 15;
+    return A[0];
+}
```

Comment:
We need to avoid using/copying 3rd party code, can we use the random-number generator from the large-dataset example here?

Reply:
Updated